restli-common/src/main/java/com/linkedin/restli/common/validation/CreateOnly.java

```diff
@@ -33,7 +33,8 @@
  * CreateOnly fields can be specified in a create request but not in a partial update request.
  * They should be specified in update requests, but they should have the same value as the original entity
  * (if an optional field was missing from the entity, it should be missing in the update request too).
- * <b>This is not checked by the Rest.li framework and should be checked in the resource implementation.</b>
+ * <b>This is not checked by the Rest.li framework, but checked by the {@RestLiValidationFilter}, if it is enabled.
```

Comment:
Saying "this is not checked by the framework" is misleading, since RLVF is arguably part of the framework. I would say it's not validated _by default_ in Rest.li, but such validation can be enabled by adding RLVF.

Reply:
Updated